qualtran/bloqs/chemistry/thc_notebook_utils.py

```diff
@@ -79,27 +80,11 @@ def generalize(bloq):
         return attrs.evolve(bloq, angle=phi_sym)
     if isinstance(bloq, And):
         return attrs.evolve(bloq, cv1=and_cv0, cv2=and_cv1)
+    if isinstance(bloq, SelectSwapQROM):
```

Comment:
This throws and error: AttributeError: 'QROM' object has no attribute 'gate', I wonder if this is necessary following #428

Reply:
fixed the error.